cmd/handler.go

```diff
@@ -64,10 +81,10 @@ func (h Handler) Merge(w io.Writer) error {
 	for _, n := range names {
 		err := enc.Encode(mfs[n])
 		if err != nil {
-			return err
+			log.Error(err)
+			return
 		}
 	}
 
-	return nil
-
+	return
```

Comment:
is this required?

Reply:
Indeed it isnt. Removed that unnecessary return